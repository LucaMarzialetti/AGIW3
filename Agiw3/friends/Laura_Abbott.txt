--url_fb https://www.facebook.com/laura.abbott.796
--friends
VM25214:3 Lucy Bewley
VM25214:3 Lisa Ralph
VM25214:3 Emma Child-Williams
VM25214:3 Rebecca Hurst Richards
VM25214:3 Tina Wilson
VM25214:3 Sadie Amor
VM25214:3 Karen Simpson
VM25214:3 Lois Griffin
VM25214:3 Jo Burbidge
VM25214:3 Amy Bush
VM25214:3 Leiha Bartlett Heywood
VM25214:3 Sara Ramsden
VM25214:3 Mark J Chandler
VM25214:3 Jade Korinna Brough
VM25214:3 Lisa Thorne
VM25214:3 Gemma Davis
VM25214:3 Kieran Fereday
VM25214:3 Laura Jayne Edwards
VM25214:3 Katie Chandler
VM25214:3 Kerry Pomphrey
VM25214:3 Phil Reed
VM25214:3 Helen Taylor
VM25214:3 Kelly Crew
VM25214:3 Laura Louise
VM25214:3 Matthew Williams
VM25214:3 Zoey Kearns
VM25214:3 Stacey Gunning
VM25214:3 Stacey Jones
VM25214:3 Cherie McNally
VM25214:3 Julie Shone
VM25214:3 Mark Kearns
VM25214:3 Bertie Wiltshire
VM25214:3 Harriet Ives
VM25214:3 Gemma Hanson
VM25214:3 Fliss Webb
VM25214:3 Andrea Steel
VM25214:3 Kaylee Smith
VM25214:3 Lisa Rose
VM25214:3 Karen Ross
VM25214:3 Tash Herriott
VM25214:3 Tracey Louise Allen
VM25214:3 Laura Swann
VM25214:3 Kelly Bamber
VM25214:3 Christine Southwood
VM25214:3 Andy Weller
VM25214:3 Becky Tomlin
VM25214:3 Jan Bolton
VM25214:3 Viv Pearce
VM25214:3 David Wilson
VM25214:3 Kerry Evans
VM25214:3 Gemma Keith
VM25214:3 Jamie Rogers
VM25214:3 Amy Claire Duckett
VM25214:3 Francesca Hateley
VM25214:3 Katherine George
VM25214:3 Emma Lacey
VM25214:3 Kate Hudson
VM25214:3 Jonathan Siggins
VM25214:3 Katie Hallett
VM25214:3 Sue Slater
VM25214:3 Laura Jane Mortimer
VM25214:3 Christopher Biggs
VM25214:3 Dave Griffin
VM25214:3 James M Chandler
VM25214:3 Leanna Lock
VM25214:3 Julie Payne
VM25214:3 Naomi Jasmine
VM25214:3 Gemma Dyer
VM25214:3 Rachel Abbott
VM25214:3 Tracey Ann Higgins
VM25214:3 Michelle Howard
VM25214:3 Natalie Hylands
VM25214:3 Adam Slusarczyk
VM25214:3 Marie Stanley
VM25214:3 Ashley Brolia
VM25214:3 Kerry Bonser
VM25214:3 Natalie Hinton
VM25214:3 Tia Perry
VM25214:3 Carole Tanner
VM25214:3 Katy Teagle
VM25214:3 Jillian Davies
VM25214:3 Naomi McKinley
VM25214:3 Ash Robinson
VM25214:3 Louisa Paul
VM25214:3 Jessica Beattie
VM25214:3 Kelly Lian Millican
VM25214:3 Laura Whitbread
VM25214:3 Perry Tracey
VM25214:3 Kirsty Fogg
VM25214:3 Darren Brewer
VM25214:3 Sheree Ramsden
VM25214:3 Shane Millarvie
VM25214:3 Nigel Amor
VM25214:3 Lindsay Gadd
VM25214:3 Karen Young
VM25214:3 Emma Lee
VM25214:3 Mandy Johnson
VM25214:3 Tracey Jayne Perry
VM25214:3 Alan Davis
VM25214:3 Michelle Hallas
VM25214:3 Ben Gadd
VM25214:3 Kaiya Marie
VM25214:3 Sharon Golding
VM25214:3 Mandy Gillard
VM25214:3 Mike Crockett
VM25214:3 Chris Duggan
VM25214:3 Cheryl Louise Bailey
VM25214:3 Sharleene Weeks
VM25214:3 Holly Kemmery
VM25214:3 Scott Stowell
VM25214:3 Christine Price
VM25214:3 Neil Brewer
VM25214:3 CJ Cameron
VM25214:3 Sharon Portsmouth
VM25214:3 Clare Watson
VM25214:3 Kate Ives
VM25214:3 Darren Smith
VM25214:3 Jenny Bourton
VM25214:3 Amy Parry
VM25214:3 Lou Parfitt
VM25214:3 David Jones
VM25214:3 Danni Isaacs
VM25214:3 Janine Smith
VM25214:3 Anton Wilkes
VM25214:3 Tasha Propert
VM25214:3 Lisa Smith
VM25214:3 Zara Wnek Coelho
VM25214:3 Steve Cross
VM25214:3 James Peach
VM25214:3 Keli Ponting
VM25214:3 Luke Turvey
VM25214:3 Jenna S G R
VM25214:3 Mark Pearcey
VM25214:3 Holly Larcombe
VM25214:3 Paul Jones
VM25214:3 Lisa Adams
VM25214:3 Rachael Hardwick-Evans
VM25214:3 Becky Hicks
VM25214:3 Paul Patton
VM25214:3 Ruth Dellow
VM25214:3 Garry Heywood
VM25214:3 Lorraine Strange
VM25214:3 Laura Swann
VM25214:3 Lee Bishop
VM25214:3 Wendy Peacock
VM25214:3 Robert Williams
VM25214:3 Charlotte Natt
VM25214:3 Kirsty Scott
VM25214:3 Laura Pearce
VM25214:3 Laura Rodney
VM25214:3 Esther Ruth Hayward
VM25214:3 Adam Natt
VM25214:3 Molly Bradley
VM25214:3 Dominic Bailey-Clay
VM25214:3 Gary Davis
VM25214:3 Anthony Timbrell
VM25214:3 Gifted Melksham
VM25214:3 Kayleigh Tennant
VM25214:3 John Robinson
VM25214:3 Andy Healey
VM25214:3 Scott Holland
VM25214:3 James White
VM25214:3 Steps Ahead Nursery
VM25214:3 Julie Gasper
VM25214:3 Clare Jones
VM25214:3 Diana Malling
VM25214:3 Thomas Mears
VM25214:3 Laura Saunders
VM25214:3 Lois Griffin Interiors
VM25214:3 Cassie Smith
VM25214:3 Kirsty Ayliffe
VM25214:3 Chelly Plumb
VM25214:3 Lynda White
VM25214:3 Laura Service
VM25214:3 Mistie Cook
VM25214:3 Nick Wellman
VM25214:3 Stacey Hackett
VM25214:3 Jess White
VM25214:3 Christopher Burnett
VM25214:3 Chris Stokes
VM25214:3 Jen Hardy
VM25214:3 Jen Teagle
VM25214:3 Simon Bush
VM25214:3 Kimberley Tagman
VM25214:3 Richard Branton
VM25214:3 Sarah Church
VM25214:3 Jenna Skedgell
VM25214:3 Sharon Clark
VM25214:3 Peter Dobbin
VM25214:3 Emma Short
VM25214:3 Dave Stockley
VM25214:3 Lorna Blake
VM25214:3 Jeremyscott Dunne
VM25214:3 Ronnie Boyer
VM25214:3 Pauline Tessier
VM25214:3 Alex Tanner
VM25214:3 Kerri Al Bshara
VM25214:3 Simon Lewis
VM25214:3 Tiff Wiltshire
VM25214:3 Zarah Wright
VM25214:3 Emlyn Lewis
VM25214:3 Sheryll Gail Holley
VM25214:3 Abigail Holley
VM25214:3 Claire Winks
VM25214:3 Dave Drewitt
VM25214:3 Matt Hooper
VM25214:3 Chris Usher
VM25214:3 Tracey Griffin
VM25214:3 Jerri Tyson
VM25214:3 Lee Bishop
VM25214:3 Chris Needham
VM25214:3 Hannah Harrison
VM25214:3 Craig Blizzard
VM25214:3 LilMiss Pixie Dust
VM25214:3 Paul David Owen
VM25214:3 Keith Melton
VM25214:3 Lou Connerty
VM25214:3 Rachael Pearce
VM25214:3 Alice Adcock - Costa
VM25214:3 Ryan Minshall
VM25214:3 Claire Herbert
VM25214:3 Scott Fleming
VM25214:3 Paul McGleish
VM25214:3 Shelley Inman
VM25214:3 Lee Dalton
VM25214:3 Jessica Matthews
VM25214:3 Kelly Bailey
VM25214:3 Rebecca Harrold
VM25214:3 Dan Hince
VM25214:3 Vicky Lewis
VM25214:3 Rachael Harlowe
VM25214:3 Owen Holland
VM25214:3 Julie Wells
VM25214:3 Kate Yabou
VM25214:3 Carl Abbott
VM25214:3 Kate Yabou
VM25214:3 Lisa Cole
VM25214:3 Eric Whitfield
VM25214:3 Clare Louisa Manning
VM25214:3 Matthew Oopsie Bridges
VM25214:3 Amy Louise
VM25214:3 Jonathan Scott
VM25214:3 Richard Gardner
VM25214:3 Tim Norris
VM25214:3 Natasha Wood
VM25214:3 Mike Barbieri
VM25214:3 Vicky Butcher Sartain
VM25214:3 David Neil
VM25214:3 Helen Hookway
VM25214:3 Laura Wakeman
VM25214:3 Vicks Hunt
VM25214:3 Laura Marie Jenkins
VM25214:3 Anthony Cook
VM25214:3 Kayley Woollacott
VM25214:3 Rae Tagg
VM25214:3 Neil Holley
VM25214:3 Tom Reggo Reynolds
VM25214:3 Steve Noble
VM25214:3 Michael Bell
VM25214:3 Julie Hallett
VM25214:3 Nick Strange
VM25214:3 Stacie Amanda Courtley
VM25214:3 Chris Griffiths
VM25214:3 Tanya Perrett
VM25214:3 David Wood
VM25214:3 Mike Rivett
VM25214:3 Suzanne Morris
VM25214:3 Katy Abbott
VM25214:3 Wendy Peacock
VM25214:3 Sophie Rush
VM25214:3 George Lovelock
VM25214:3 Pete J Findley
VM25214:3 Wayne Potter
VM25214:3 Love Furnishing
VM25214:3 Lloyd Studley
VM25214:3 Emma Ellis
VM25214:3 Alex Shapley
VM25214:3 Roxy Sales
VM25214:3 Alisha Waugh
VM25214:3 Tracey Lynn Abbott
VM25214:3 Catherine Wilson
VM25214:3 Jesique Nails
VM25214:3 Emma Farley
VM25214:3 Graham Moorhouse
VM25214:3 Gill Bourne
VM25214:3 Gary Edmunds
VM25214:3 Suzie George
VM25214:3 Gemma Louise
VM25214:3 Sam Bonavia
VM25214:3 Scarlett Angel Gifts Christie
VM25214:3 Jazz Selling
VM25214:3 Sam Louise Miller
VM25214:3 Alex Coombs
VM25214:3 Shirl Osborne
VM25214:3 Chris Griffiths
VM25214:3 Lee Fordham
VM25214:3 The-BodyShop Laura
VM25214:3 Kidz Clozet
VM25214:3 Samantha Jayne Morrison
VM25214:3 Gary Holland
VM25214:3 Bethan Eurfron Holley
VM25214:3 Natasha Lake
VM25214:3 Lucy Lyons
VM25214:3 Andy Freeman
VM25214:3 Joanne Cleverley
VM25214:3 Aaron Smith
VM25214:3 David Dyke
VM25214:3 Rhidian Holley
VM25214:3 Donna Sparks
VM25214:3 Jonny Douglas
VM25214:3 Mark Schuler
VM25214:3 Jamie Goss
VM25214:3 Togetherness We Will Fight
VM25214:3 Vicky Thomas
VM25214:3 Anthony Wilcox
VM25214:3 Adrian Tipper
VM25214:3 Robin Higgins
VM25214:3 Laura Dommett
VM25214:3 Alex Haste
VM25214:3 Liz Hughes
VM25214:3 Luke Adams
VM25214:3 Alec Price
VM25214:3 Thomas Joshua Peare-Harte
VM25214:3 Matthew Peter Bridges
VM25214:3 Anthony White
VM25214:3 Andy Viveash
VM25214:3 Richie Butler
VM25214:3 Colin Clarke
VM25214:3 Carrie Heathcote
VM25214:3 Luke Foster
VM25214:3 Suzanne Mary Ramsden
VM25214:3 Sarah Garland
VM25214:3 Gary Miller
VM25214:3 Pamela Bryant
VM25214:3 Cara Petty
VM25214:3 Carole Ekwah
VM25214:3 Mikhala Melville
VM25214:3 Dale Beattie
VM25214:3 Katherine Mills
VM25214:3 Alex Pratley
VM25214:3 Tim Howard
VM25214:3 Vicky James