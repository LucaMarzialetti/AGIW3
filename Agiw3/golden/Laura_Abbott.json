{
    "PATTERN": {
        "tel": [],
        "addr": [],
        "email": []
    },
    "NER": {
        "LOC": ["Waterford"],
        "ORG": [
            "Facebook",
            "Twitter",
            "LinkedIn"
        ],
        "PER": [
            "Dr. Laura Abbott",
            "family physician",
            "Diabetic Retinopathy",
            "Personal Trainer"
        ]
    },
    "url": "http://www.healthchoicesfirst.com/practitioner/dr-laura-abbott-family-doctor-waterford-nb"
}